packages/resource-deployment/templates/README.md

```diff
@@ -0,0 +1,55 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+# Services dashboard
+
+Time settings, including the overall time range and granularity of aggregations, can be changed in UTC time button in the top left.
+
+### Dashboard tiles:
+
+-   **Availability**
+    -   (# Availability test failures) / (# Availability test runs)
+    -   Availability tests are run every 5 minutes by the health-monitor-timer-func in web-workers. The test verifies that we can:
+        1. Ping the health endpoint to check that the service is available
```

Comment:
Can you also add information on the responsibility on each function? 
timer, orchestrator & activity trigger?

Reply:
Added to the web-workers README